seadssite/views.py

```diff
@@ -41,7 +41,7 @@ def register(request):
 
     # if a user "POST"s, check phone number, provider and user/profile form(models)
     if request.method == 'POST':
-        print ("user is about to register")
+        print("user is about to register")
```

Comment:
Logging can be really helpful. Can we replace these print statements with proper logging that includes a timestamp and preferably the logging location?